snap/local/helper-go/configure.go

```diff
@@ -581,26 +567,25 @@ func configure() {
 		// check core services
 		startServices, err = checkCoreConfig(startServices)
 		if err != nil {
-			hooks.Error(fmt.Sprintf("edgexfoundry:configure: core config error; %v", err))
-			os.Exit(1)
+			log.Fatalf("core service config error: %v", err)
 		}
 
 		// check optional services
 		startServices, err = checkOptConfig(startServices)
 		if err != nil {
-			hooks.Error(fmt.Sprintf("edgexfoundry:configure: optional config error; %v", err))
-			os.Exit(1)
+			log.Fatalf("optional service config error: %v", err)
 		}
 
-		// NOTE - the services will be started after the configure hook finishes
-		if err = cli.StartMultiple(true, startServices...); err != nil {
-			hooks.Error(fmt.Sprintf("edgexfoundry:configure failure starting/enabling services: %v", err))
-			os.Exit(1)
+		for i, s := range startServices {
+			startServices[i] = hooks.SnapName + "." + s
```

Comment:
why does each service need to include hooks.SnapName?

Reply:
That's because the service name take the form `<snap-name>.<app-name>`. Here, the services slice actually contains the app names.